spec_2/scrabble.js

```diff
@@ -0,0 +1,57 @@
+const playedTiles = "AERETOXMYCNS_B"
+
+const HIGHEST_TILE_COUNT = 12
+
+const tiles = {
+  A: { count: 9 },
+  B: { count: 2 },
+  C: { count: 2 },
+  D: { count: 4 },
+  E: { count: 12 },
+  F: { count: 2 },
+  G: { count: 3 },
+  H: { count: 2 },
+  I: { count: 9 },
+  J: { count: 1 },
+  K: { count: 1 },
+  L: { count: 4 },
+  M: { count: 2 },
+  N: { count: 6 },
+  O: { count: 8 },
+  P: { count: 2 },
+  Q: { count: 1 },
+  R: { count: 6 },
+  S: { count: 4 },
+  T: { count: 6 },
+  U: { count: 4 },
+  V: { count: 2 },
+  W: { count: 2 },
+  X: { count: 1 },
+  Y: { count: 2 },
+  Z: { count: 1 },
+  _: { count: 2 }
+}
+const tileFaces = [ 'A','B','C','D','E','F','G','H','I','J','K',
```

Comment:
Also, I tend not to use values as keys. My reasoning is, if you consider that each of these entries is a `Tile`, what are attributes of a `Tile`?  Is an attribute `A` or `letter`?  I would argue that the information here is named `letter`, and the value for an individual `Tile` might be `A`.  

As well, using dynamically named keys tends to make logic harder later (for more involved programs; you don't run into the problem in this small program).